applications/mne_anonymize/fiffanonymizer.cpp

```diff
@@ -716,282 +227,445 @@ int FiffAnonymizer::censorTag(FiffTag::SPtr outTag,
-        outTag->resize(newStr.size());
-        memcpy(outTag->data(),newStr.toUtf8(),static_cast<size_t>(newStr.size()));
-        printIfVerbose("Project Persons changed: " + QString(inTag->data()) + " -> " + newStr);
+        QString inStr(m_pTag->data());
+        QString outStr(m_sProjectPersons);
+        m_pTag->resize(outStr.size());
+        memcpy(m_pTag->data(),outStr.toUtf8(),static_cast<size_t>(outStr.size()));
+        printIfVerbose("Project persons changed: " + inStr + " -> " + outStr);
         break;
     }
     case FIFF_PROJ_COMMENT:
     {
-        if(m_bBruteMode) {
-            QString newStr(m_sProjectComment);
-            outTag->resize(newStr.size());
-            memcpy(outTag->data(),newStr.toUtf8(),static_cast<size_t>(newStr.size()));
-            printIfVerbose("Project comment changed: " + QString(inTag->data()) + " -> " + newStr);
+        if(m_bBruteMode)
+        {
+            QString inStr(m_pTag->data());
+            QString outStr(m_sProjectComment);
+            m_pTag->resize(outStr.size());
+            memcpy(m_pTag->data(),outStr.toUtf8(),static_cast<size_t>(outStr.size()));
+            printIfVerbose("Project comment changed: " + inStr + " -> " + outStr);
         }
         break;
     }
     case FIFF_MRI_PIXEL_DATA:
     {
-        qWarning() << " ";
-        qWarning() << "The input fif file contains MRI data.";
-        qWarning() << "Beware that a subject''s face can be reconstructed from it";
-        qWarning() << "This software can not anonymize MRI data, at the moment.";
-        qWarning() << " ";
+        printIfVerbose("  ");
+        printIfVerbose("Warning: The input fif file contains MRI data.");
+        printIfVerbose("Warning: Beware that a subject''s face can be reconstructed from it");
+        printIfVerbose("Warning: This software can not anonymize MRI data, at the moment.");
+        printIfVerbose("  ");
         break;
     }
     default:
+    {   }
+    }
+}
+
+inline QString FiffAnonymizer::subjectSexToString(const int sexCode) const
+{
+    static QStringList subjectSexRefList = { "unknown" , "male" , "female" };
+    if (sexCode > -1 && sexCode < subjectSexRefList.size())
+    {
+        return subjectSexRefList.at(sexCode);
+    } else {
+        qCritical() << "Invalid subject sex code. [0 = unknown, 1 = male, 2 = female]. The code of the subject is: " << QString::number(sexCode);
+        return QString("invalid-code");
+    }
+}
+
+inline QString FiffAnonymizer::subjectHandToString(const int handCode) const
+{
+    static QStringList subjectHandRefList = { "unknown", "right", "left", "ambidextrous" };
+    if ((handCode > -1) && (handCode < subjectHandRefList.size()))
+    {
+        return subjectHandRefList.at(handCode);
+    } else {
+        qCritical() << "Invalid subject handedness code. [0 = unknown, 1 = right, 2 = left, 3 = ambidextrous]. The code of the subject is: " << QString::number(handCode);
+        return QString("invalid-code");
+    }
+}
```

Comment:
add newline

Reply:
I think you mean to unpack those lists. Done that.

Reply:
no I mean a new line before the `//====================` spacer